### DISCUSSION TITLE
EN depot export string to Arkplanner needs adjusted (ark-nights site does not import)

### DISCUSSION THREAD

**Discussion by NanaKG** [MaaAssistantArknights/MaaAssistantArknights#9668] [General]
From what I can tell the export string has English names which breaks the import for the ark-nights site. The import on ark-nights only detects the name in Chinese(except RMA70-12/24, I assume it must be that way on Chinese client?) 

P.S Only know English, Tried search for similar discussions with ArkPlanner(用数据) translated, none of them seemed to mention EN so don't think this EN client problem is mentioned.

**Comment by ABA2396** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/9672
